core/lib/workarea/configuration.rb

```diff
@@ -1321,6 +1321,13 @@ def self.setup_defaults
         cvv
         amount
       ]
+
+      # This is the Array of Symbols used to determine whether
+      # `Order#add_item` will create a new `Order::Item` record or
+      # whether it will update an existing one. Attributes from
+      # `Catalog::Customizations` are always included in this existing
```

Comment:
As written, these docs aren't true, customizations are handled in a separate method. Let's make this description true.

Reply:
Ah yeah, I wanted to be clear that customizations are always checked for equivalence in a separate method, but I wasn't sure how to say that here.